.github/workflows/native_publish_pytorch.yml

```diff
@@ -7,6 +7,10 @@ on:
           description: 'staging/snapshot, default is snapshot'
           required: true
           default: 'snapshot'
+        package_type:
```

Comment:
Rather than making this an input, it might be better to run as a matrix and have both package builds in one run?

Reply:
The problem is on sonatype side, in gradle we are publishing each package sequentially, ideally, each package should landed in the same repo or each with its own repo. We don't have control the close the repo for each package. Even we manually run two publish command, it may or may not in the same repo. Currently, I manually publish one, login to sonatype, close the repo and run another publish.